kroma-bindings/predeploys/addresses.go

```diff
@@ -79,7 +79,7 @@ func IsProxied(predeployAddr common.Address) bool {
 func init() {
 	Predeploys["ProxyAdmin"] = &oppredeploys.Predeploy{Address: ProxyAdminAddr}
 	Predeploys["WETH9"] = &oppredeploys.Predeploy{Address: WETH9Addr, ProxyDisabled: true}
-	Predeploys["L1Block"] = &oppredeploys.Predeploy{Address: L1BlockAddr}
```

Comment:
Why not rename the L1Block in other predeployments to KromaL1Block for consistency?